### ISSUE TITLE
Hello,Login to Meross Cloud,An unexpected error occurred

### ISSUE THREAD

**Issue by Jennie324** [albertogeniola/meross-homeassistant#511]
**Describe the bug**
Login to Meross Cloud
An unexpected error occurred

**Your environment**
HomeAssistant version: -- 2024.8.1 --
Hassio Version (if applicable): -- 1.34.0 --
Hardware environment: -- arm64-armbian-docker --

**Logs taken when the issue happened**
[
<img width="411" alt="iShot_2024-08-15_16 30 36" src="https://github.com/user-attachments/assets/9d2a5f5d-106e-4888-9308-a53c539784f8">
<img width="394" alt="iShot_2024-08-15_16 27 39" src="https://github.com/user-attachments/assets/7276a613-f471-4eba-8d1a-66a1260fd0e9">
](url)

**Comment by dennishoeting** [comment]
Same issue here, +1

**Comment by manur37** [comment]
Same issue, I tested with all the server addresses mentioned ! Always the same mistake.
No double identification on my account and no special characters in the password. The connection from the Meross app works well.

**Comment by Richspr** [comment]
HI I also have the same issue when trying to sign to connect

**Comment by g4v4g4i** [comment]
Same here.

**Comment by rajBar** [comment]
Same issue here too. Coincided with me being logged out of the meross app too

**Comment by 2006ciro** [comment]
same

**Comment by wesleyharkins1093** [comment]
Same issue here - Any resolutions?

**Comment by gregborn** [comment]
Same issue, two devices

Misery loves company?

**Comment by tristanbes** [comment]
Same :)
Changed password on my account, but still got an error even after password reset

**Comment by varactor** [comment]
Same issue here, 15 unusable devices :/

**Comment by HerrVonundZu** [comment]
Same here. Will Switch to local plugin for now

**Comment by meza** [comment]
Same issue here

**Comment by marcunson** [comment]
same

**Comment by NadirKamran** [comment]
Same here!

**Comment by dpajanssen** [comment]
Same here!

**Comment by Wozzagonebush** [comment]
Same here. First got logged out of the Android app, then all my devices stopped working.

Managed to log back into Android app but no joy with the integration.

**Comment by jpearn** [comment]
Has anyone had the following email before it stopped working ?

_This is meross cloud security team contacting you.

First of all, thanks for choosing meross products and we hope that you enjoy smart life with our smart products.

Recently our cloud security system has noticed that your devices are communicating with the could server at an extremely high frequency. We consider these behaviours abnormal and are concerned about the security of your devices.

Due to the security concerns and the intent to avoid any further potential damage, we will terminate the cloud services of these devices for now to protect your devices.

If you are sure that your devices are safe, NOT hacked, and are performing as expected. Please first lower your periodic request rate to no more than one message every ten seconds and then send an email to <[support@meross.com](mailto:support@meross.com)>. We will resume cloud service for you within 24 hours.

For the safety of your device and personal data, we suggest you to use meross app to manage your devices and DO NOT use any other third-party platforms. Please note that any abnormal high cloud request frequency will cause cloud service termination again.

Should you have any information you would like to provide us with or you would like to offer us any clarification, please contact us within 24 hours._

**Comment by Mick-Ma** [comment]
Yes, I had that email. I emailed them back about having too many of their products which is why they see so many communications and next day it worked again.

**Comment by wesleyharkins1093** [comment]
I never saw that email. Been an annoying week without all my integrations working.
Sent from my iPhone

On Aug 24, 2024, at 12:21 PM, Mick-Ma ***@***.***> wrote:

﻿

Yes, I had that email. I emailed them back about having too many of their products which is why they see so many communications and next day it worked again.

**Comment by jasonwitty** [comment]
hey guys see my post on this thread.

https://github.com/albertogeniola/meross-homeassistant/issues/510

I would imagine that the issue with your account getting locked by the meros engineers is because your integrations are continuing to poll over and over without a sucessful login. So it could be a symptom and not the root cause. the API was changed to remove a field and i dont think there is any way this exact code base could work with the api right now.

**Comment by rajBar** [comment]
> hey guys see my post on this thread.
> 
> #510
> 
> I would imagine that the issue with your account getting locked by the meros engineers is because your integrations are continuing to poll over and over without a sucessful login. So it could be a symptom and not the root cause. the API was changed to remove a field and i dont think there is any way this exact code base could work with the api right now.

WORKED A CHARM! You're a legend, thank you!

**Comment by Richspr** [comment]
Hi not working for me. I tried your suggestions in post thread #510 _
I could not find a pycache folder in my meross cloud files.
I have deleted and re installed by still get the same unexpected error occurred. 
I only have one device so guess I am going to have to do with out it at the moment

**Comment by dalanmiller** [comment]
I am receiving the same thing. 

When I go to https://iot.meross.com or https://iotx-ap.meross.com in the browser I see: 

```
{"error_msg":"request too frequent"}
```

Despite getting an HTTP 200 😓. So at least for me this is self-explanatory. However, I am noticing that even after using a VPN that I'm getting the same message, so either they have somewhat sophisticated client recognition or the host is down for everyone?
